client/app/pages/settings/OrganizationSettings.jsx

```diff
@@ -155,6 +155,15 @@ class OrganizationSettings extends React.Component {
             ))}
           </Select>
         </Form.Item>
+        <Form.Item label="Multi-byte Search">
+          <Checkbox
+            name="multi_byte_search_enabled"
+            checked={formValues.multi_byte_search_enabled}
+            onChange={e => this.handleChange('multi_byte_search_enabled', e.target.checked)}
+          >
+          Enable multi-byte search for query names and descriptions (slower)
```

Comment:
How about the following description?

```suggestion
          Enable multi-byte (Chinese, Japanese, and Korean) search for query names and descriptions (slower)
```

It might not be 100% technically accurate, but will be more familiar to users.

Reply:
Thank you, will update this soon!